<ui version="4.0" >
 <comment>
***********************************************************************
*
* Copyright (c) 2021-2022 Barbara Geller
* Copyright (c) 2021-2022 Ansel Sermersheim
*
* Copyright (c) 2015 The Qt Company Ltd.
*
* Designer is free software. You can redistribute it and/or
* modify it under the terms of the GNU Lesser General Public License
* version 2.1 as published by the Free Software Foundation.
*
* Designer is distributed in the hope that it will be useful,
* but WITHOUT ANY WARRANTY; without even the implied warranty of
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
*
* https://www.gnu.org/licenses/
*
***********************************************************************
 </comment>
 <class>qdesigner_internal::TableWidgetEditor</class>
 <widget class="QDialog" name="qdesigner_internal::TableWidgetEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Table Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="itemsTab" >
      <attribute name="title" >
       <string>&amp;Items</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget" >
            <property name="toolTip" >
             <string>Table Items</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="buttonsLayout" >
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="showPropertiesButton" >
              <property name="text" >
               <string>Properties &amp;>></string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qdesigner_internal::TableWidgetEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>431</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel" >
     <x>373</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qdesigner_internal::TableWidgetEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>547</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel" >
     <x>562</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
